src/internal/operations/backup_test.go

```
@@ -361,265 +556,106 @@ func (suite *BackupOpSuite) TestBackupOperation_ConsumeBackupDataCollections_Pat
+				{
+					Manifest: manifest1,
+					SubtreePaths: []*path.Builder{
+						emailBuilder,
+					},
+				},
+			},
+		},
+		{
+			name: "SingleManifestMultipleReasons",
+			inputMan: []*kopia.ManifestEntry{
+				{
+					Manifest: manifest1,
+					Reasons: []kopia.Reason{
+						emailReason,
+						contactsReason,
+					},
+				},
+			},
+			expected: []kopia.IncrementalBase{
+				{
+					Manifest: manifest1,
+					SubtreePaths: []*path.Builder{
+						emailBuilder,
+						contactsBuilder,
+					},
+				},
+			},
+		},
+		{
+			name: "MultipleManifestsMultipleReasons",
+			inputMan: []*kopia.ManifestEntry{
+				{
+					Manifest: manifest1,
+					Reasons: []kopia.Reason{
+						emailReason,
+						contactsReason,
+					},
+				},
+				{
+					Manifest: manifest2,
+					Reasons: []kopia.Reason{
+						emailReason,
+						contactsReason,
+					},
+				},
+			},
+			expected: []kopia.IncrementalBase{
+				{
+					Manifest: manifest1,
+					SubtreePaths: []*path.Builder{
+						emailBuilder,
+						contactsBuilder,
+					},
+				},
+				{
+					Manifest: manifest2,
+					SubtreePaths: []*path.Builder{
+						emailBuilder,
+						contactsBuilder,
+					},
+				},
+			},
+		},
 	}
 
-	return res
-}
+	for _, test := range table {
+		suite.T().Run(test.name, func(t *testing.T) {
+			ctx, flush := tester.NewContext()
+			defer flush()
 
-func makeManifest(backupID model.StableID, incompleteReason string) *snapshot.Manifest {
-	backupIDTagKey, _ := kopia.MakeTagKV(kopia.TagBackupID)
+			mbu := &mockBackuper{
+				checkFunc: func(
+					bases []kopia.IncrementalBase,
+					cs []data.Collection,
+					service path.ServiceType,
+					oc *kopia.OwnersCats,
+					tags map[string]string,
+					buildTreeWithBase bool,
+				) {
+					assert.ElementsMatch(t, test.expected, bases)
+				},
+			}
 
-	return &snapshot.Manifest{
-		Tags: map[string]string{
-			backupIDTagKey: string(backupID),
-		},
-		IncompleteReason: incompleteReason,
+			//nolint:errcheck
```

Swallowing this error is going to influence troubleshooting for CI. Is this temporary?

---

This code isn't changed in the PR: you're looking at bad diffs due to conflicts.